sdcm/cluster.py

```diff
@@ -3145,14 +3145,21 @@ def destroy(self):
         for node in self.nodes:
             node.destroy()
 
-    def terminate_node(self, node):
+    def terminate_node(self, node, scylla_shards=""):
```

Comment:
better not to pass scylla_shards to terminate node and not to change the current method signature. Please find a way to avoid passing scylla_shards

Reply:
Done